### DISCUSSION TITLE
Is there any better guide available for using Vuex?

### DISCUSSION THREAD

**Discussion by amirsafaricg** [quasarframework/quasar#11473] [Ideas / Proposals] [answered]
Hi guy's 

is there any tutorial or guide for using Vuex in quasar ? 
I can't use vuex based on the guide

**Comment by rstoenescu** [comment]
What's wrong with the guide?

**Comment by MilosPaunovic** [comment]
The [guide](https://quasar.dev/quasar-cli/vuex-store) is only for making VUEX work in Quasar. You could always refer to VUEX docs for use cases.

**Reply by amirsafaricg** [reply]
the problem is I dont know how to define multiple mutations for my states in this format of usage :
`export const updateDrawerState = (state) => {
  state.drawerState = !state.drawerState;
};
`

**Reply by SergioAndreev** [reply]
> the problem is I dont know how to define multiple mutations for my states in this format of usage : `export const updateDrawerState = (state) => { state.drawerState = !state.drawerState; }; `

did you find the solution?

**Comment by yusufkandemir** [comment, marked as answer]
> the problem is I dont know how to define multiple mutations for my states in this format of usage :
`export const updateDrawerState = (state) => { state.drawerState = !state.drawerState; };`

It turned out that the [OP actually wants to know how to define multiple mutations](https://github.com/quasarframework/quasar/discussions/11473#discussioncomment-1700484), and here is the answer:
```js
// The guide shows a similar mutation
export const toggleDrawerState = (state) => {
  state.drawerState = !state.drawerState;
};

// So, just add more of them if you want to add more of them, ? :)
export const resetWhatever = (state) => {
  state.whatever = 'hehehe';
};

// With a payload
export const setWhatever = (state, value) => {
  state.whatever = value === 'hehehe' ? value : value.toLowerCase();
};
```
